bench/profiles/projects.yml

```diff
@@ -26,7 +26,7 @@ charts:
 
 scripts:
     dotty:
-        - measure -feature -deprecation -unchecked -Xfatal-warnings -encoding UTF8 -language:existentials,higherKinds,implicitConversions,postfixOps -Yexplicit-nulls -with-compiler $(find $PROG_HOME/dotty/compiler/src/dotty -name *.scala -o -name *.java)
+        - measure -feature -deprecation -unchecked -Xfatal-warnings -encoding UTF8 -language:existentials,higherKinds,implicitConversions -Yexplicit-nulls -with-compiler $(find $PROG_HOME/dotty/compiler/src/dotty -name *.scala -o -name *.java)
```

Comment:
You could also remove existentials and higherKinds (here and in Build.scala) since they don't do anything in scala 3.

Reply:
higherKinds also does nothing in scala 2.

Reply:
I know, I'm the one who silenced it :) https://github.com/scala/scala/pull/8348